CMakeLists.txt

```diff
@@ -11,37 +11,18 @@ list(GET _VERSION_LIST 2 MICRO_VERSION)
 
 set(CMAKE_PROJECT_VERSION ${MAJOR_VERSION}.${MINOR_VERSION}.${MICRO_VERSION})
 
-
-###############################################################################
-##  From:
-##    https://cliutils.gitlab.io/modern-cmake/chapters/projects/submodule.html
-##
-##  Use git to add, initialize, and update submodules for the user
-###############################################################################
-find_package(Git QUIET)
-if(GIT_FOUND AND EXISTS "${PROJECT_SOURCE_DIR}/.git")
-    # Update submodules as needed
-    option(GIT_SUBMODULE "Check submodules during build" ON)
-    if(GIT_SUBMODULE)
-        message(STATUS "** Submodule update")
-        execute_process(COMMAND ${GIT_EXECUTABLE} submodule update --init --recursive
-                        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-                        RESULT_VARIABLE GIT_SUBMOD_RESULT)
-        if(NOT GIT_SUBMOD_RESULT EQUAL "0")
-            message(FATAL_ERROR "git submodule update --init failed with ${GIT_SUBMOD_RESULT}, please checkout submodules")
-        endif()
-    endif()
-endif()
-
 ################################################################################
 # TODO:  Add licensing and authorship information
 # TODO:  Test with FEtk build
 # TODO:  Handle special mac dependencies
 #        (configure.ac:1306)
 ################################################################################
-set(CMAKE_VERBOSE_MAKEFILE true)
-cmake_policy(SET CMP0054 OLD)
+#set(CMAKE_VERBOSE_MAKEFILE false)
 cmake_policy(SET CMP0042 NEW)
```

Comment:
@nsoblath can you document what each of these policies do and why we have them set they way we do?

Reply:
Done (see de954f23d8b48b2b9edd11c94eaf27f4bfd8741c)